### PR TITLE
feat(wes): create Lit package for WES

### LINKED ISSUE #148: Template the package and add create Runs TES component

### PR DESCRIPTION
**Pull request by JaeAeich**
## Description
Create a package for lit migration for WES and add Create run component

Fixes #148

### CODE REVIEW

**Comment by JaeAeich** [comment]
@git-anurag-hub This is wesCreate, the fields and form are set. Was able to create workflows, need to figure out how to show response though but maybe that part of design package implementation.

I think we should maybe merge it and then work on those things namely:
- Reposponse 
- styles

**Comment by git-anurag-hub** [comment]
Yeah make sense @JaeAeich, we need to implement 2/3 more functionalities to the form component like: 
- success() method
- error() method
- loading() method

I will create the issue for the same!

**Comment by uniqueg** [comment]
@JaeAeich can you please tell me on Slack what exactly you need, in terms of a workflow?

**Comment by JaeAeich** [comment]
@git-anurag-hub Please review this.

**Comment by JaeAeich** [comment]
@git-anurag-hub Please review it. @uniqueg you can try to create workflow from the [deployement](https://cloud-components-git-lit-wes-elixir-cloud-aai.vercel.app/wes/components/wes-create-run.html), do check it out if you can.

**Comment by uniqueg** [comment]
This looks awesome! :star_struck: Great job @JaeAeich & @git-anurag-hub.

And the test workflow worked great for me! :rocket:

I do have, of course, a couple of points, but they shouldn't block this PR. You can of course include them here, if they are easy to do. Otherwise, maybe create issues for them to tackle later:

- It would be great to have descriptions or tool tips for the different params. For example, workflow URL can be used to reference an uploaded file, as per the spec (though that may currently not be implemented in proWES). Noboody would be able to know that unless they do know the spec. Of course the question is, where to source these descriptions/tool tips from - but that we could discuss in the issue (probably just hardcoding them would be best; they probably won't change often, if at all).
- I think the `tags` field is missing (it [is defined in the `1.0.1` version of the specs](https://github.com/ga4gh/workflow-execution-service-schemas/blob/13f56821fb50e4b01a269354421c96c20785e909/openapi/workflow_execution_service.openapi.yaml#L192-L193), which we should minimally support)
- I would reorder the fields to have as follows (from top to bottonm): "Workflow URL", "Workflow Type", "Workflow Type Version", "Workflow Parameters". The reason is that to me it seems to me more natural that I would want to decide which workflow I'm running _first_, before worrying about any params or the engine. For that I _may_ need to select the files first, then reference the main descriptor file, say what type and type version of descriptor it is. After all of that I would put the "Tags" and then finally the "Workflow Engine Parameters". Note that in WES v1.1.0, we will have two more (optional) workflow engine-related parameters (workflow engine name and version), so at that point I think it would make sense to collapse them.
- I would only capitalize the first word for the field names, e.g., "Workflow type" instead of "Workflow Type"
- For the workflow attachments, it would be nice (and I think also important) to see the names of the files that have been uploaded, especially since it may be necessary to select one of them as the "Workflow URL". And I think it is a better user experience if the user can delete individual files, and if a new selection of files does not clear the old selection. So for example, I might select a primary CWL descriptor file and two secondary descriptor files. But accidentally I have selected a wrong file. So I should be able to remove the wrong file and then select only the single missing descriptor file, without having to select again the two others.
- As an extension of the issue above, it would actually be great if after selecting one or more files for upload and listing their file names and a button to remove them, there would also be a checkbox that would optionally allow selecting exactly _one_ file as the Workflow URL. If selected, this should auto-populate (and ideally hidden or grayed out unless a file is unselected) the "Workflow URL" field with the name of the file. As an alternative (possibly even better) we could maybe have an upload file button _next to_ the "Workflow URL" field (maybe with an `OR` in between) that can only be used to select the primary descriptor file. If it is used, the workflow URL is then auto-populated. Additional files can then still be attached through the button/field at the bottom.
- It would be nice if the "Workflow Parameters" and "Workflow Engine Parameters" would also accept YAML (which would then be converted to JSON), as it is easier for humans to manipulate YAML. For example, `{"input":{"class":"File","path":"https://is.muni.cz/pics/design/r/znak_MU.png"}}` is much harder to type/construct than:

   ```yaml
   input:
     class: File
     path: https://is.muni.cz/pics/design/r/znak_MU.png
   ```

- Finally, I think it would be good to highlight the "Submit" button (bold font, colored background) and gray it out until all required values have been added.

**Comment by JaeAeich** [comment]
@git-anurag-hub I have opened issues for the features stated above (#172 , #173, #174, #175). Please review the issues as well as they are feat/fixes for design package, we might need to see if the solution should be abstracted or left to `pacakge-author`.

File: apps/documentation/docs/wes/installation.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Installation
+
+`@elixir-cloud/wes` can be installed using npm as
+
+```bash
+npm install @elixir-cloud/lit-wes
+```
```
[the remark this answers is not in the document]
Comment (JaeAeich) [reply apps/documentation/docs/wes/installation.md]: done :+1:

**Comment by JaeAeich** [comment]
> For the time being, as a workaround for file multiple input selection, you can utilize the array structure and add the files as children to a single selection field. This approach will display all the file names, and users will be able to delete individual files. However, the downside is that users will need to click the "Add" button each time they want to include a new file. Considering the advantages of this implementation, I believe this workaround is reasonable. @uniqueg can confirm whether to go ahead with the implementation.

Sounds awesome though I think maybe the current implementation is ok! as we can have multiple inputs acc specs, we shouldn't change it else we will have to change the logic too. If its fine we can wait and finish TES (which requires more attention) first and add multiple selection later.
just a suggestion though. we have a workaround as you mentioned. If it gets too late we can always implement it.

**Comment by uniqueg** [comment]
Thanks @git-anurag-hub. I actually think that currently, we do not absolutely need the file support, because even if we can see the uploaded files listed and remove individual files and so on, it will still be a blocker that subdirectory structures cannot be preserved with the current implementation. In real world use cases, all required files will almost never be in a single flat directory. See my new [comment](https://github.com/elixir-cloud-aai/cloud-components/issues/173#issuecomment-1807908297) for details.

So I would rather leave it as it is for now, until we can address it properly.

**Comment by uniqueg** [comment]
Please also note that I have added two more issues: #179 & #180

**Pull request merged**